lib/rubocop/cop/rails/active_record_calculation.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Enforces the use of ActiveRecord calculation methods instead of `pluck`
```

Comment:
Let's add the rationale here: it avoids loading potentially many values into memory by doing the calculations through the equivalent database function.

Reply:
Good idea 👍